src/agent/api/timelineapi.ts

```diff
@@ -7,10 +7,14 @@ import restm = require('./restclient');
 
 export class TimelineApi implements ifm.ITimelineApi {
```

Comment:
This is all generated now so we don't want to change this - contact Teddy Ward for that to land first